build-system/build.conf.js

```diff
@@ -26,9 +29,70 @@ const defaultPlugins = [
   require.resolve('./babel-plugins/babel-plugin-transform-version-call'),
 ];
 
+/**
+ * @return {Array<string|Object>} the minify-replace plugin options that can be
+ * pushed into the babel plugins array
+ */
+function getReplacePlugin() {
+  /**
+   * @param {string} defineStr the define flag to parse
+   * @return {!Object} replacement options used by minify-replace plugin
+   */
+  function createReplacement(defineStr) {
+    const strSplit = defineStr.split('=');
```

Comment:
Is this to support `--defineExperimentConstant=EXP_A=true,EXP_B=false`. Just curious does `=` works well in the argv?

Reply:
Yes, this is to support `--defineExperimentConstant=EXP_A=true`. So you can set EXP_A to any value